src/app/request-copy/grant-request-copy/themed-grant-request-copy.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { ThemedComponent } from 'src/app/shared/theme-support/themed.component';
+import { GrantRequestCopyComponent } from './grant-request-copy.component';
+
+/**
+ * Themed wrapper for grant-request-copy.component
+ */
+@Component({
+  selector: 'ds-themed-grant-request-copy',
+  styleUrls: [],
+  templateUrl: './../../shared/theme-support/themed.component.html',
```

Comment:
This path looks wrong.. it starts with a single dot?

I think this should be '../../shared/theme-support/themed.component.html'

Reply:
@mwoodiupui : It is possible this will work... I just found the `./` to be odd looking & unnecessary.  But, you may be correct that the behavior is the same either way.